xbmc/input/ButtonTranslator.h

```diff
@@ -78,6 +78,8 @@ class CButtonTranslator
   static void GetActions(std::vector<std::string> &actionList);
   static void GetWindows(std::vector<std::string> &windowList);
 
+  bool IsPossibleLongpress(int window, const CKey &key);
```

Comment:
BTW should we change the method name to be more suitable.

`IsKeyLongPressConfigured(int window, const CKey &key)`